server/import.go

```diff
@@ -89,6 +115,38 @@ func createPublic(private tpm2.Private, hashAlg tpm2.Algorithm, pcrs *tpmpb.Pcrs
 			Unique: publicHash.Sum(nil),
 		},
 	}
+	setPublicAuth(&public, pcrs)
+	return public
+}
+
+func createPrivatePublicRSA(rsaPriv *rsa.PrivateKey, nameAlg tpm2.Algorithm, pcrs *tpmpb.Pcrs) (tpm2.Private, tpm2.Public, error) {
```

Comment:
We should use public then private for consistancy with other methdos

Reply:
We could make this:

`createPublicPrivateSign(priv crypto.PrivateKey) -> (tpm2.Public, tpm2.Private, error)`

This method would then do the appropriate casting of crypto.PrivateKey to the correct type. This also makes adding ECC support easier.

Reply:
A helper function like this should be moved to `key_conversion.go` to avoid cluttering `import.go`